src/components/RouterComponent/RouterComponent.tsx

```diff
@@ -1,12 +1,14 @@
 import React from "react";
 import { Navigate, Route, Routes } from "react-router-dom";
 import CarList from "../../pages/CarList/CarList";
+import CompareCars from "../../pages/CompareCars/CompareCars";
 
 function RouterComponent() {
   return (
     <div>
       <Routes>
         <Route path="/" element={<CarList />}></Route>
+        <Route path="/comparecars" element={<CompareCars />}></Route>
```

Comment:
Let's stick to kebab-case for path names
```suggestion
        <Route path="/compare-cars" element={<CompareCars />}></Route>
```